tests/index.test.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable max-lines-per-function */
+
 const mockfs = require('mock-fs');
```

Comment:
@gwenf Make sure that all comments are inline so they can be replied to directly. Even if it's just in a random file on a random line.

This file is using `mock-fs`, you can us it as an example. I am not able to reproduce your issue. The current commited code runs identically with `npm t` in Windows and Ubuntu.